### PR TITLE
README:  project info, contributions, sprint information & roadmap

### PR DESCRIPTION
**Pull request by alexander-san**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,126 @@
...
+
+
+# Tests
+
+right now there is only a very limited amount of tests
+
+This will be changed in Sprint 3
+
+``` bash
+code for running the single line command in the command line goes here
```
Comment (Jmion) [inline README.md]: What do you mean with a single line command?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,126 @@
...
+code example goes here
+```
+
+## Installation
+
+For installing this project in the development environment you will need:
+
+- Gradle
+- Java v.1.8
+- Android SDK v.28
```
Comment (Jmion) [inline README.md]: technically our min SDK is 15. See gradle app file. The target is what we will run on if it is available. Our app sould work on SDK 15

File: README.md (hunk 1 above)
Comment (alexander-san) [reply README.md]: Automated test suite: Your app must have from day one a test suite that can be executed by running a single command on the command line. You must maintain a minimum level of 80% code coverage.
http://dslab.epfl.ch/teaching/sweng/proj

Comment (alexander-san) [reply README.md]: Also please don't block the PR if you don't understand something, that is why there is a 3. button called, comment but approve

**Pull request merged**